sphinx/source/docs/user_guide/extensions.rst

```diff
@@ -162,7 +162,7 @@ then the complete Python class might look like:
 
     class Custom(LayoutDOM):
 
-        __implementation__ = open("custom.coffee").read()
+        __implementation__ = "custom.coffee"
```

Comment:
`__implementation__` accepts a string or an instance implementing `Implementation` interface. Accepting string is a convenience measure and, by default, nothing has changed here. A string is considered CoffeeScript source code. However, if a string is a one-liner which ends in `.coffee`, `.js`, `.css` or `.less`, then it will be interpreted as a path (relative to file where a model is defined). This new convenience feature shouldn't conflict with any earlier code.